llvm/lib/Target/VE/MCTargetDesc/VEInstPrinter.cpp

```diff
@@ -172,15 +148,7 @@ void VEInstPrinter::printMemASOperandRRM(const MCInst *MI, int OpNum,
 
 void VEInstPrinter::printMemASOperandHM(const MCInst *MI, int OpNum,
                                         const MCSubtargetInfo &STI,
-                                        raw_ostream &O, const char *Modifier) {
-  // If this is an ADD operand, emit it like normal operands.
-  if (Modifier && !strcmp(Modifier, "arith")) {
```

Comment:
This change LG. VE doesn't seem to use modifier in AsmString

Reply:
Thanks, resolving